### PR TITLE
Contact information

### PR DESCRIPTION
**Pull request by JakobMiksch**
Adds two functions regarding the contact information of the GeoServer.

- `getContactInformation`
- `updateContactInformation`

### CODE REVIEW

File: test/test.js [hunk 1]
```diff
@@ -21,6 +21,29 @@ describe('Basic GeoServer', () => {
...
+    const country = 'Deutschland';
+    const postalCode = '123445';
+    const state = 'Berlin';
+    const email = 'example email address';
+    const organization = 'A organization';
+    const contactPerson = 'My contact persion' ;
+    const phoneNumber = '1231234234123';
+    
+    let result = await grc.settings.updateContactInformation(address, city, country, postalCode, state, email, organization, contactPerson, phoneNumber);
+    expect(result).to.be.true;
```
Comment (chrismayer) [inline test/test.js:43]: Wouldn't it make sense here to query the settings again and check if the values match as additional expects?

**Review by chrismayer — APPROVED** [review]
LGTM, thanks! I'd suggest to address my comments but leave this up to you.

File: test/test.js (hunk 1 above)
Comment (JakobMiksch) [reply test/test.js:43]: Good point. I added a test like this.

**Comment by JakobMiksch** [comment]
@chrismayer I applied your comment and adjusted the docs.

**Review by chrismayer — APPROVED** [review]
LGTM, thanks for this @JakobMiksch

**Pull request merged**